Methods/Parameters/Inventory/Inventory/Program.cs

```diff
@@ -0,0 +1,314 @@
+
+    if(option == 1)
+    {
+        foundItem.Quantity = value;
+    }
+    else
+    {
+        foundItem.Price = value;
+    }
+
+
+
+    Console.WriteLine($@"Item successfully updated.");
+
+    ReturnToMenu();
+
+    
+}
+void RemoveItem()
+{
+    Console.WriteLine(@"
+----------------------------Remove Item----------------------------
+Please enter the name of the item you want to remove:");
+
+    string findname = FindItem();
+
+    Console.WriteLine(@$"
+Are you sure you want to remove {findname} from the inventory? (Y / N) :
+
+--------------------------------------------------------------------");
+    string confirmation = Console.ReadLine().ToUpper();//needs to be a method
+    while (string.IsNullOrWhiteSpace(confirmation) || confirmation != "Y" && confirmation != "N")
+    {
+        Console.WriteLine("Invalid input. Please enter Y/N.");
+        confirmation = Console.ReadLine();
+    }
+
+    if ( confirmation == "Y")
+    {
+        items.RemoveAll(i => i.Name == findname);
+        Console.WriteLine($@"Item successfully removed.");
+    }
+    else
+    {
+        Console.WriteLine("Item was not removed.");
+    }
+
+    ReturnToMenu();
+}
+string FindItem()
+{
+    string findname ="";
+    bool itemnotfound = true;
+    while (itemnotfound)
+    {
+
+        findname = Console.ReadLine();
+        while (string.IsNullOrWhiteSpace(findname))
+        {
+            Console.WriteLine("Invalid input. Please enter a valid name.");
+            findname = Console.ReadLine().ToUpper();
+        }
+        foreach (Item i in items)
+        {
+            if (i.Name == findname)
+            {
+                itemnotfound = false;
+                
+            }
+
+
+        }
+
+        if (itemnotfound) { 
+        Console.Clear();
+        Console.WriteLine("Name not found. Try again.");
+        }
+    }
+    return findname; ;
+}
+void AddItemDisplay()
+{ //static (name, int, int) AddItemDisplay(){}
+    Console.WriteLine(@"
+        ----------------------------Add New Item ----------------------------
+    Please enter the item name:");
+    string name = Console.ReadLine();
+    while (string.IsNullOrWhiteSpace(name))
+    {
+        Console.WriteLine("Invalid input. Please enter a valid name.");
+        name = Console.ReadLine().ToUpper();
+    }
+    Console.WriteLine(@"
+    Please enter the item quantity:");
+    int quantity;
+    while (!int.TryParse(Console.ReadLine(), out quantity) || quantity > 100)
+    {
+        Console.WriteLine("Invalid input. Please enter a valid number for quantity:");
+    }
+    Console.WriteLine(@"
+    Please enter the item price per unit:");
+    int price;
+    while (!int.TryParse(Console.ReadLine(), out price) || price > 100)
+    {
+        Console.WriteLine("Invalid input. Please enter a valid number for price:");
+    }
+    Additem(name, quantity, price);
+}
+void Additem(string name, int quantity, int price) {
```

Comment:
The method name Additem does not follow PascalCase. Use consistent casing for all method names.
Rename the method to AddItem